app/common/expressions/managed.py

```diff
@@ -41,5 +43,9 @@ def message(self) -> str:
 
     @property
     def expression(self) -> str:
-        # todo: are UUIDs parsable by the expression parser/ language
-        return f"(( {self.question_id} )) > {self.minimum_value}"
+        # todo: how much of these should be calculated from the context and run
+        #       through the eval vs. stored in the expression - this is leaning towards the latter
+        # todo: are UUIDs parsable by the expression language
```

Comment:
I _think_ these have slipped back in from a rebase and are largely answered by this PR - ideally they could be removed. The final todo could either stay or leave up to yo